### ISSUE TITLE
Support ASCII output

### ISSUE THREAD

**Issue by grouville** [goccy/go-graphviz#53]
Hello everyone,
Thanks for this awesome library. Do you plan on adding support to ASCII output anytime ?

cc [Dagger discussion](https://github.com/dagger/dagger/discussions/1587#discussioncomment-2157093)

**Comment by goccy** [comment]
Yes. We are thinking of supporting if there is demand. By the way, the link destination seems to be 404.

**Comment by goccy** [comment]
@grouville @jlongtine  From the latest version, you can now add custom renderers. Please give it a try !
